funtofem/driver/fun3d_oneway_driver.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         self.struct_name = struct_name
 
     @classmethod
-    def paths(cls, fun3d_dir, aero_name="fun3d", struct_name="struct"):
+    def paths(cls, fun3d_dir, aero_name="fun3d", struct_name="tacs"):
```

Comment:
Can you add documentation on output_name, aero_name, and struct_name to the parameters section in Fun3dRemote.__init__?